src/TileSlider/TileSlider.tsx

```diff
@@ -209,6 +220,9 @@ const TileSlider = <T extends unknown>({
         document.removeEventListener('touchmove', handleTouchMove);
         document.removeEventListener('touchend', handleTouchEnd);
         document.removeEventListener('touchcancel', handleTouchCancel);
+
+        if (verticalScrollBlockedRef.current) verticalScrollBlockedRef.current = false;
```

Comment:
Always reset to `false`:

```suggestion
        verticalScrollBlockedRef.current = false;
```